pkg/upgrade/upgrade_state_test.go

```diff
@@ -1047,6 +1121,122 @@ var _ = Describe("UpgradeStateManager tests", func() {
 			Expect(getNodeUpgradeState(node)).ToNot(Equal(upgrade.UpgradeStateDone))
 		})
 	})
+	It("UpgradeStateManager should not move outdated node to UpgradeRequired states with orphaned pod", func() {
+		orphanedPod := &corev1.Pod{}
+
+		UnknownToUpgradeRequiredNode := nodeWithUpgradeState("")
+		DoneToUpgradeRequiredNode := nodeWithUpgradeState(upgrade.UpgradeStateDone)
```

Comment:
Shouldn't this variable be named `DoneToUpgradeDoneNode`?

Reply:
Done